src/components/Card/CardWithPicture1.jsx

```diff
@@ -0,0 +1,220 @@
+import React, { useState } from "react";
```

Comment:
@mohdhaider07 `CardWithPicture1.jsx` and `CardWithoutPicture1.jsx` aren't being used in Credentials Tests. 

Please remove this from this files as it uses the mui v4.